cfme/tests/automate/test_method.py

```diff
@@ -348,3 +350,99 @@ def test_send_email_method(smtp_test, klass):
     # TODO(GH-8820): This issue should be fixed to check mails sent to person in 'cc' and 'bcc'
     # Check whether the mail sent via automate method really arrives
     wait_for(lambda: len(smtp_test.get_emails(to_address=mail_to)) > 0, num_sec=60, delay=10)
+
+
+@pytest.fixture(scope="module")
+def generic_definition(appliance):
+    # Creating generic object using REST
+    with appliance.context.use(ViaREST):
+        definition = appliance.collections.generic_object_definitions.create(
+            name="LoadBalancer_{}".format(fauxfactory.gen_numeric_string(3)),
+            description="LoadBalancer",
+            attributes={"location": "string"},
+            associations={"vms": "Vm", "services": "Service"}
+        )
+        yield definition
+        if definition.exists:
+            definition.delete()
+
+
+@pytest.fixture
+def service_req(request, generic_catalog_item):
+    # Creating generic service
+    service_request = generic_catalog_item.appliance.rest_api.collections.service_templates.get(
+        name=generic_catalog_item.name
+    ).action.order()
+    request.addfinalizer(service_request.action.delete)
+
+
+@pytest.mark.tier(1)
+def test_automate_generic_object_service_associations(appliance, klass, service_req,
+                                                      generic_definition):
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/10h
+        caseimportance: medium
+        startsin: 5.7
+        casecomponent: Automate
+
+    Bugzilla:
+        1410920
+    """
+    # Ruby code
+    script = 'go_class = $evm.vmdb(:generic_object_definition).find_by(:name => "{name}")\n'.format(
+        name=generic_definition.name
+    )
+    script = script + (
+        'load_balancer = go_class.create_object(:name => "Test Load Balancer", :location => '
+        '"Mahwah")\n'
+        '$evm.log("info", "XYZ go object: #{load_balancer.inspect}")\n'
+        'service = $evm.vmdb(:service).first\n'
+        'load_balancer.services = [service]\n'
+        'content_type = "message"\n'
+        'load_balancer.save!\n'
+        '$evm.log("info", "XYZ load balancer got service: #{load_balancer.services.first.inspect}")'
```

Comment:
Should this be "XYZ load balancer go service" ?

Reply:
I have used this code directly from BZ and I think it is written for understanding in logs.
Please suggest your thoughts if you want it in other way :)

Reply:
How about replacing `XYZ` with `test_automate_generic_object_service_associations` ? More easy to find in logs, IMO.

Reply:
I don't think, we need to change this and I am not getting how it will  be easy to find in log by changin it to ```test_automate_generic_object_service_associations``` because at the end we will grep or search the string we want in logs.

Reply:
It would make it less ambiguous, because one cannot be sure if there will be another entry in the logs including `XYZ` but can be more sure that your test name won't be logged by anything apart from your test. It is just a suggestion.